skimage/data/generate_shapes.py

```diff
@@ -0,0 +1,316 @@
+import numpy as np
+
+from .. import draw
+from .._shared.utils import warn
+
+import collections
+
+Label = collections.namedtuple('Label', 'category, x1, x2, y1, y2')
+Point = collections.namedtuple('Point', 'row, column')
+ShapeProperties = collections.namedtuple('ShapeProperties',
+                                         'min_size, max_size, color')
+ImageShape = collections.namedtuple('ImageShape', 'nrows, ncols, depth')
+
+
+def _generate_rectangle_mask(point, image, shape, random):
+    """Generate a mask for a filled rectangle shape.
+
+    The height and width of the rectangle are generated randomly.
+
+    Parameters
+    ----------
+    point : Point
+        The row and column of the top left corner of the rectangle.
+    image : ImageShape
```

Comment:
We don't usually employ named tuples to enforce shape shapes.  Thoughts, @scikit-image/core ?

Reply:
I agree in general, but since this function is internal only, I would be fine with the current implementation. It makes the list of parameters a little more compact.

Reply:
I would be in favor in more style consistency (plain tuples instead of named tuples) with the rest of the codebase, with the hope of more readable code for developers.